mbus/nats_handler.go

```diff
@@ -130,17 +130,35 @@ func (h *natsHandler) Start(handlerFunc boshhandler.Func) error {
 	h.logger.Info(h.logTag, "Attempting to connect to NATS")
 
 	if net.ParseIP(connectionInfo.IP) != nil {
-		err = h.platform.DeleteARPEntryWithIP(connectionInfo.IP)
-		if err != nil {
-			h.logger.Error(h.logTag, "Cleaning ip-mac address cache for: %s", connectionInfo.IP)
-		}
+		h.arpClean()
 	}
-
 	var natsOptions = []nats.Option{
-		nats.Timeout(natsConnectionTimeout),
+		nats.DisconnectHandler(func(c *nats.Conn) {
```

Comment:
Why we use the deprecated `DisconnectHandler` and not the replacement `DisconnectErrHandler? See https://github.com/nats-io/nats.go/pull/504/files

Reply:
Nice natch. I haven't seen the deprecation notice. 
Thanks.